tests/components/manual/test_alarm_control_panel.py

```diff
@@ -151,16 +152,16 @@ async def test_arm_home_with_invalid_code(hass):
     assert hass.states.get(entity_id).state == STATE_ALARM_DISARMED
 
 
-async def test_arm_away_no_pending(hass):
-    """Test arm home method."""
+async def test_arm_home_with_template_code(hass):
```

Comment:
Why are these tests changed?

Reply:
I did not change them, just reordered one to group all test for home mode. I can revert this if needed

Reply:
no more relevant since tests refactoring